### PR TITLE
Implement ExternalTokenDetailPage

### PR DESCRIPTION
**Pull request by toshiakitakase**
- Implement ExternalTokenDetailPage

Sample URL
/dashboard/0xD92E713d051C37EbB2561803a3b5FBAbc4962431

Note:
The display of the Template header button in Storybook is incorrect.

### CODE REVIEW

File: frontend/packages/react-app/src/components/templates/ExternalTokenDetailPageTemplate/index.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      description:
+        "Audiusアカウントのフォロワーの方へトークンを配布します。手順はXXX...",
+      image: "https://example.com/campaignimage.jpg",
+    },
+    claimAmount: "0",
+    distributor: {
+      id: "0x48889feca4574810e5a5b30b6b93146a837500fb",
+    },
+    id: "0xcc22b3199013627708d05fedf9b854bd7706a156",
+    startDate: "20201212",
```
Comment (wild-mouse) [inline frontend/packages/react-app/src/components/templates/ExternalTokenDetailPageTemplate/index.stories.tsx:43]: Is this value from subgraph result?
If it is, value of date is timestamp such as `1612137600`. 
https://thegraph.com/explorer/subgraph/tart-tokyo/iroiro-rinkeby?selected=playground

Comment (toshiakitakase) [reply frontend/packages/react-app/src/components/templates/ExternalTokenDetailPageTemplate/index.stories.tsx:43]: Ok, I will change sample data to timestamp.
Current campaign startDate return null from subgraph.

Comment (toshiakitakase) [reply frontend/packages/react-app/src/components/templates/ExternalTokenDetailPageTemplate/index.stories.tsx]: Fixed

**Review by wild-mouse — APPROVED** [review]
LGTM 🏃‍♂️

**Pull request merged**